build.sbt

```diff
@@ -54,3 +54,6 @@ inThisBuild(
     )
   )
 )
+
+addCommandAlias("applyCodeStyle", ";+clean ;scalafixAll ;scalafmtAll; scalafmtSbt")
```

Comment:
The `+clean` is here since scalafix uses scala compiler and if there is dirty state then it can only check code that is covered by the incremental compiler.

Also `scalafixAll` is run before `scalafmtAll`/`scalafmtSbt` since its possible that if scalafix applies some code changes, they then need to be formatted.